app/views/application/_error_summary.html+govwales.erb

```diff
@@ -0,0 +1,14 @@
+<% if flash[:validation]&.any? %>
+  <%# GOVWALES: 'govuk-width-container' addition. Lines 3 & 13 %>
```

Comment:
Don't need the comment now we're using the override template?

Reply:
See below next comment. Ditto.

Reply:
Removed.